src/config.ts

```diff
@@ -3,15 +3,22 @@ import type { TypedDataSigner } from '@ethersproject/abstract-signer';
 import SafeServiceClient from "@gnosis.pm/safe-service-client";
 import CeramicClient from "@ceramicnetwork/http-client";
 import { IDX } from "@ceramicstudio/idx";
+import WalletConnect from "@walletconnect/client";
 import config from "@app/config.json";
+import { WalletConnectSigner } from "./WalletConnectSigner";
 
 declare global {
   interface Window {
     ethereum: any;
     registrarState: any;
   }
 }
+/// Gas limits for various transactions.
+const gasLimits = {
+  createOrg: 1_200_000,
+};
 
+let walletConnect: WalletConnect;
```

Comment:
I would avoid global mutable state. This should go inside the `Config` class

Reply:
throws an error if i put it inside the config class

Reply:
As a class attribute I don't see why it would cause problems..

Reply:
Unresolved :warning:

Reply:
this has been resolved